src/pydvl/value/shapley/gt.py

```diff
@@ -227,15 +230,15 @@ def group_testing_shapley(
     n = len(u.data.indices)
     const = _constants(
         n=n,
-        epsilon=eps,
-        delta=0.05,
+        epsilon=epsilon,
+        delta=delta,
         utility_range=u.score_range.max() - u.score_range.min(),
```

Comment:
Should we raise an error if group testing is used but no range is passed?

Reply:
Or maybe we can just not try to compute a value for `max_iterations` and leave it to the user. I'll warn instead